Provis.Core/Services/WorkspaceService.cs

```diff
@@ -157,8 +157,8 @@ public async Task DenyInviteAsync(int id, string userid)
                 throw new HttpException(System.Net.HttpStatusCode.BadRequest, "You cannot deny this invite");
             }
 
-            if(inviteUserRec.IsConfirm==null)
-            inviteUserRec.IsConfirm = false;
+            if (inviteUserRec.IsConfirm == null)
+                inviteUserRec.IsConfirm = false;
```

Comment:
Should we use null propagation ?

Reply:
Thats not my piece of code, i just accidentally move the string. I will transfer the information to developer